README.md

```diff
@@ -1,64 +1,65 @@
-	- [PCDM metadata Model](#Newspapers-PCDM-metadata-model)
-- [Installation/Testing](#installationtesting)
-	- [Extending, Using](#extending-using)
-	- [Basic Model Use (console)](#basic-model-use-console)
-	- [Application/Site Specific Configuration](#applicationsite-specific-configuration)
-	- [Development and Testing Setup](#development-and-testing-setup)
-- [Credits](#credits)
-	- [Sponsoring Organizations](#sponsoring-organizations)
-	- [Contributors and Project Team](#contributors-and-project-team)
-	- [More Information / Contact](#more-information-contact)
-<!-- /TOC -->
-# Introduction
-The Newspapers in Samvera is an IMLS grant funded project to develop newspaper specific functionality for the [Samvera](http://samvera.org/) Hyrax framework.
+  - [Documentation](#documentation)
+  - [Requirements](#requirements)
+  - [Dependencies](#dependencies)
+- [Installation](#installation)
+  - [Application/Site Specific Configuration](#applicationsite-specific-configuration)
+    - [Config changes made by the installer:](#config-changes-made-by-the-installer)
+    - [Configuration changes you should make after running the installer](#configuration-changes-you-should-make-after-running-the-installer)
+- [Ingesting Content](#ingesting-content)
+- [Developing, Testing, and Contributing](#developing-testing-and-contributing)
+  - [Contributing](#contributing) 
+- [Acknowledgements](#acknowledgements)
+  - [Sponsoring Organizations](#sponsoring-organizations)
+  - [More Information](#more-information)
+  - [Contact](#contact)
 
-## Documentation
-We are currently working on adding and updating documentation on our [Project Wiki](https://github.com/marriott-library/newspaper_works/wiki)
+<!-- /TOC -->
 
 # Overview
-The Newspaper Works gem provides work type models and administrative functionality for Hyrax-based Samvera applications in the space of scanned newspaper media.  This gem can be included in a Digital Asset Management application based on Hyrax 2.5.1
+NewspaperWorks is a gem (Rails "engine") for [Hyrax](http://hyrax.samvera.org/) -based digital repository applications to support ingest, management, and display of digitzed newspaper content.
 
-## Purpose, Use, and Aims
-This gem, while not a stand-alone application, can be integrated into an application based on Hyrax 2.5 easily to support a variety of cases for management, ingest, and archiving of primarily scanned (historic) newspaper archives.
+NewspaperWorks is not a stand-alone application. It is designed to be integrated into a new or existing Hyrax (2.5.x) application, providing content models, ingest workflows, and feature-rich UX for newspaper repository use-cases.
 
-## Development Status
-This gem is currently under development. The development team is actively working on this project and is updating the codebase nightly. We are targeting an initial 1.0 release for June 2019.
+NewspaperWorks supports:
+* models for Title, Issue, Page, and Article
+* batch ingest via command line
```

Comment:
Do we want to specifically mention via rake task?

Reply:
I think this is OK as is, this is the super-condensed feature list, don't think we need a ton of detail here?